cmd/neofs-adm/internal/modules/morph/dump_hashes.go

```diff
@@ -140,6 +138,37 @@ func dumpCustomZoneHashes(cmd *cobra.Command, nnsHash util.Uint160, zone string,
 		})
 	}
 
+	sessionID, iter, err := unwrap.SessionIterator(inv.Call(nnsHash, "tokens"))
+	if err != nil {
+		if errors.Is(err, unwrap.ErrNoSessionID) {
+			cmd.PrintErrln("Provided RPC endpoint doesn't seem to support sessions, some hashes might be lost.")
```

Comment:
You may print it if `len(items) == nnsMaxTokens` below, otherwise we have all of them anyway.

Reply:
Would be nice, fixed